osu.Game.Rulesets.Mania/UI/Stage.cs

```diff
@@ -134,12 +135,14 @@ public Stage(int firstColumnIndex, StageDefinition definition, ref ManiaAction c
             {
                 bool isSpecial = definition.IsSpecialColumn(i);
 
-                var column = new Column(firstColumnIndex + i, isSpecial)
+                var action = columnStartAction;
+                columnStartAction++;
+                var column = CreateColumn(firstColumnIndex + i, isSpecial).With(c =>
                 {
-                    RelativeSizeAxes = Axes.Both,
-                    Width = 1,
-                    Action = { Value = columnStartAction++ }
-                };
+                    c.RelativeSizeAxes = Axes.Both;
+                    c.Width = 1;
+                    c.Action.Value = action;
+                });
```

Comment:
I'm only *sorta* confident the `columnStartAction` changes here are correct. Remember, friends don't let friends use postfix increment in complex expressions. (Ideally ever.)